frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -491,7 +433,8 @@ ReaderFooter.default_settings = {
     text_font_bold = false,
     container_height = G_defaults:readSetting("DMINIBAR_CONTAINER_HEIGHT"),
     container_bottom_padding = 1, -- unscaled_size_check: ignore
-    progress_margin_width = Screen:scaleBySize(Device:isAndroid() and material_pixels or 10), -- default margin (like self.horizontal_margin)
+    progress_margin_width = Device:isAndroid() and Screen:scaleByDPI(16) or 10, -- android: guidelines for rounded corner margins
+    progress_margin = false,
```

Comment:
Been wondering while reading the whole thing what this settings does. As (I guess) it's saved and read from existing settings you can't change its name, may be add a comment here ?

Reply:
https://github.com/koreader/koreader/blob/856233e711adcea3c2cd9d3ffd0fc3a0ede757fe/frontend/apps/reader/modules/readerfooter.lua#L2189

Reply:
I've read it a few times before I would have reached that comment :)
May be move/duplicate that comment, and be more precise about its type:
`-- true if progress bar margins same as book margins `